ui/src/app/cohort-review/set-annotation-create/set-annotation-create.component.html

```diff
@@ -7,11 +7,12 @@
         class="tooltip tooltip-validation tooltip-sm tooltip-bottom-left"
         [class.invalid]="name.invalid && (name.dirty || name.touched)">
         <input id="name"
-               class="annotation-name-input"
+          class="annotation-name-input"
           name="name"
           formControlName="name"
           type="text"
-               placeholder="ANNOTATION NAME"
+          placeholder="ANNOTATION NAME"
+          maxlength="225"
```

Comment:
Why 225? I think the limit in the DB is technically 255...

Reply:
I will make it as 255 ... Thanks